yb-voyager/cmd/import.go

```diff
@@ -258,6 +258,11 @@ func registerImportSchemaFlags(cmd *cobra.Command) {
 		"Refreshes the materialised views on target during post snapshot import phase (default false)")
 	BoolVar(cmd.Flags(), &enableOrafce, "enable-orafce", true,
 		"enable Orafce extension on target(if source db type is Oracle)")
+
+	// --post-snapshot-import and --refresh-mviews flags will now be handled by the command post-data-import-finalize-schema
+	// Not removing these flags and just hiding them for backward compatibility.
+	cmd.Flags().MarkHidden("post-snapshot-import")
+	cmd.Flags().MarkHidden("refresh-mviews")
```

Comment:
Additionally: If someone still uses these, we can print a message saying these are deprecated and user should prefer using the new command.

Reply:
Okay I'll add something like this:
![image](https://github.com/user-attachments/assets/899abdcd-3c81-44aa-aa64-8ce5cb6d5031)